python/nano/src/bigdl/nano/tf/optimizers/test_optimizers.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2016 The BigDL Authors.

```

Comment:
you should add test file to the test directory.
see https://github.com/intel-analytics/BigDL/tree/branch-2.0/python/nano/test/tf/keras